inst/templates/ad_adex.R

```diff
@@ -51,7 +51,7 @@ ex <- ex %>%
   mutate(EXPLDOS = if_else(EXTRT == "PLACEBO", 0, 54))
 
 
-# ---- Derivations ----
+# Derivations ----
```

Comment:
Add to line 13 a row for # Load source datasets ---- to match the other templates

Reply:
Added